src/shared/directives/conditional-show/domain/conditional-show.model.spec.ts

```diff
@@ -370,7 +371,7 @@ describe('conditional-show', () => {
     });
 
     it('field mentioned in single value condition has no value', () => {
-      let sc = new ShowCondition('fieldCONTAINS"test"');
+      let sc = new ShowCondition('field CONTAINS"test"');
```

Comment:
Same as the comment above about operators not needing to have spaces before or after...

Reply:
As I mentioned in my earlier comment